components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +212,45 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
```

Comment:
i personally prefer `displayName` over `dname`
it is more descriptive and follows convention